### PR TITLE
Fixing a dependency

### PR DESCRIPTION
**Pull request by miguel-a-calles-mba**
The remapping causes issues when using monorepo tools such as rush.

See issue https://github.com/microsoft/rushstack/issues/3260

### CODE REVIEW

**Comment by miguel-a-calles-mba** [comment]
The unit tests passed.

```
$ npm run test
  198 passing (41ms)
```

File: package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
   "dependencies": {
     "@peculiar/asn1-schema": "^2.0.44",
     "@peculiar/json-schema": "^1.1.12",
-    "@typescript/lib-dom": "npm:@types/web",
+    "@types/web": "0.0.54",
```
Comment (microshine) [inline package.json:43]: Please change the version to `^0.0.54`
Remove `package-locg.json` file and update `yarn.lock` file. This project uses `yarn`

```
npm un @types/web
npm i yarn ---global
yarn add @types/web
```

**Review by microshine — CHANGES_REQUESTED** [review]
Looks like the `@types/web` works fine

Please do changes I requested, and I'll publish this fix

**Comment by amirhmoradi** [comment]
I get `Error: @typescript/lib-dom@npm:@types/web isn't supported by any available resolver` when using yarn 3.1+ from your latest release.
I believe this PR would fix my error too.
Thanks in advance.

**Comment by miguel-a-calles-mba** [comment]
@microshine, I updated the PR. Thanks.

**Comment by microshine** [comment]
I've published the new version `webcrypto-core@1.7.1`. Please try it

**Comment by miguel-a-calles-mba** [comment]
@microshine , it's working for me in the two npm packages that lever this repo.

Any luck with your issue, @amirhmoradi?

**Pull request merged**